config/config.yaml

```diff
@@ -220,4 +220,4 @@ clouds:
           # Hypershift
           # uncomment the following line if you want to install the hypershift operator
           # with CRD support for managedIdentities
-          #hypershiftAdditionalInstallArg: '--tech-preview-no-upgrade'
+          hypershiftAdditionalInstallArg: '--tech-preview-no-upgrade'
```

Comment:
This is required to create clusters with the latest hypershift-operator. 
I'm not sure if that will be needed in the rest of env (dev and cs-pr).

Reply:
Please change to:
```yaml
hypershift:
   additionalInstallArg: "--tech-preview-no-upgrade"
```